gssapi/krb5Token_test.go

```diff
@@ -43,17 +43,20 @@ func TestMechToken_newAuthenticatorChksum(t *testing.T) {
 	assert.Equal(t, b, cb, "SPNEGO Authenticator checksum not as expected")
 }
 
-func TestMechToken_newAuthenticator(t *testing.T) {
+// Test with explicit subkey generation.
+func TestMechToken_newAuthenticatorWithSubkeyGeneration(t *testing.T) {
 	creds := credentials.NewCredentials("hftsai", testdata.TEST_REALM)
 	creds.CName.NameString = testdata.TEST_PRINCIPALNAME_NAMESTRING
 	etypeID := 18
+	keyLen := 32 // etypeID 18 refers to AES255 -> 32 bytes key
```

Comment:
Small typo in comment here. Will fix later.

Reply:
Thanks, please fix the typo so that I can merge.